config/env_sample.json

```
@@ -93,19 +93,19 @@
     "/*  LTI is disabled by default": "*/",
     "STUDENT_DASHBOARD_LTI": false,
     "/*  LTI 1.3 configuration":"*/",
-    "/*  The first key of LTI_CONFIG is your Canvas domain name":"*/",
+    "/*  The first key of LTI_CONFIG is your environment specific Canvas url and not domain specific":"*/",
     "LTI_CONFIG": {
         "https://canvas.instructure.com": [
             {
                 "/*The default set of variables for LTI validation":"*/",
                 "default": true,
                 "/*LTI Dev Key from Canvas":"*/",
                 "client_id": "17700000000000111",
-                "/*fill this out for your institution.":"*/",
+                "/*fill this out environment specific and need not to be domain specific.":"*/",
```

We need only this single, specific comment for the three URLs that follow it.

```suggestion
                "/*Allowed hosts for the following 3 URLs: canvas.instructure.com, canvas.beta.instructure.com, canvas.test.instructure.com":"*/",
```